scripts/install-deps.sh

```diff
@@ -239,7 +239,11 @@ main()
             install_deps_FreeBSD
             ;;
         *)
-            echo "OS not supported."
+            fetch_and_unpack_Catch2
+            fetch_and_unpack_fmtlib
+            fetch_and_unpack_yaml_cpp
+            echo "OS $ID not supported."
+            echo "Dependencies were fetch manually and most likely libunicode will compile."
```

Comment:
Can we add a CI check for this case? Just by overriding the OS ID to something unsupported?

Reply:
Yes, we can, this is a great idea :)

Reply:
> Build / Ubuntu Linux 22.04 (Clang 11) (pull_request) Failing after 35s

Since this one is failing, I am not sure Clang version 11 is particulary interesting for us, or is it? We have way more recent Clang versions available. I just checked. At least my Ubuntu 22.04 seems to come with Clang 11 by default (i.e. if no version suffix was given during install) but also version 12, 13, 14, 15. So it's probably save to just test for the latest available one on Ubuntu 22.04. What do you think?

Reply:
I think that it is better to have this checks, so we are aware of the issues with default compilers on Ubuntu 22.04

Reply:
Ah. You upgraded from Ubuntu 20.04 CI image to 22.04. I'd dare to say we're not having this problem in Ubuntu 20.04 CI image (if we'd test for this).